packages/renderer/src/AppNavigation.svelte

```diff
@@ -47,15 +47,18 @@ function clickSettings(b: boolean) {
   {#each $navigationRegistry as navigationRegistryItem}
     <!-- This is a section -->
     {#if navigationRegistryItem.type === 'section' && navigationRegistryItem.enabled}
-      <NavSection tooltip={navigationRegistryItem.name}>
-        <svelte:component this={navigationRegistryItem.icon.iconComponent} size={iconSize} slot="icon" />
+      {@const allItemsHidden = (navigationRegistryItem.items ?? []).every(item => item.hidden)}
+      {#if !allItemsHidden}
```

Comment:
Why declaring a variable which has a single usage ?

Reply:
it looks a little bit too long to be directly a condition. I find it more readable.

Reply:
I approved so that's not blocking for me